client/pages/me/me.js

```diff
@@ -1,66 +1,98 @@
 // pages/me/me.js
-Page({
+const qcloud = require('wafer2-client-sdk');
+const app = getApp();
 
-  /**
-   * 页面的初始数据
-   */
+Page({
   data: {
-
+    personal: {},
+    chineseName: null,
+    chineseNameInputValue: null,
+    englishName: null,
+    englishNameInputValue: null,
+    phone: null,
+    phoneInputValue: null,
+    email: null,
+    emailInputValue: null,
+    btnDisable: true
   },
-
-  /**
-   * 生命周期函数--监听页面加载
-   */
-  onLoad: function (options) {
-
+  onLoad: function () {
+    this.setData({
+      personal: app.userInfo
```

Comment:
LGTM for now. But eventually we need to handle the case that user info isn't available (network issue).